### ISSUE TITLE
Error in CRA with 0.1.3, can't compile experimental syntax.

### ISSUE THREAD

**Issue by KerimG** [olton/Metro4-React#23]
Adding `metro4-react` to a `create-react-app` seems to fail like so 

```
SyntaxError: C:\Users\kg\code\metro-example\node_modules\metro4-react\src\components\tag-input\tag-input.js: Support for the experimental syntax 'classProperties' isn't currently enabled (30:14):

  28 |     }
  29 | 
> 30 |     onChange = tags => {
     |              ^
  31 |         this.props.onChange(tags);
  32 |     };
  33 | 

Add @babel/plugin-proposal-class-properties (https://git.io/vb4SL) to the 'plugins' section of your Babel config to enable transformation.
```

Just like in this issue https://github.com/olton/Metro4-React/issues/20, I suggest avoiding using experimental syntax or transpiling down to ES5 for npm releases.

**Comment by lepresk** [comment]
I'm getting same error :( help me to fix

**Comment by olton** [comment]
I work on the fix for this problem. Unfortunately, CRA authors do not want to include support for such syntax in their application. At the moment, you can use the clone from https://github.com/olton/react-scratch to create a prototype application.

**Comment by olton** [comment]
fixed in 0.1.4

**Comment by olton** [comment]
https://blog.pimenov.com.ua/2020/05/08/using-metro4-react-with-cra/
